tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -69,6 +108,21 @@ void test_simple_formatting() {
     vformat_to(back_insert_iterator{output_string}, locale::classic(), STR("f"), make_testing_format_args<charT>());
     assert(output_string == STR("f"));
 
+    output_string.clear();
+    format_to(move_only_back_inserter{back_insert_iterator{output_string}}, STR("{} {} {} {} {} {} {} {}"), true,
+        charT{'a'}, 0, 0u, 0.0, STR("s"), basic_string_view{STR("sv")}, nullptr);
+    assert(output_string == STR("true a 0 0 0 s sv 0x0"));
+
+    output_string.clear();
+    format_to(move_only_back_inserter{back_insert_iterator{output_string}}, STR("{:} {:} {:} {:} {:} {:} {:} {:}"),
+        true, charT{'a'}, 0, 0u, 0.0, STR("s"), basic_string_view{STR("sv")}, nullptr);
+    assert(output_string == STR("true a 0 0 0 s sv 0x0"));
+
+    output_string.clear();
+    format_to_n(move_only_back_inserter{back_insert_iterator{output_string}}, 300, STR("{} {} {} {} {} {} {} {}"), true,
+        charT{'a'}, 0, 0u, 0.0, STR("s"), basic_string_view{STR("sv")}, nullptr);
+    assert(output_string == STR("true a 0 0 0 s sv 0x0"));

```

Comment:
we should test a std::formatter based type as well, although it doesn't have to be in this PR. (to prove it works you could add a `static_cast<void*>(nullptr)`)